### ISSUE TITLE
Allow changing window size from config variables

### ISSUE THREAD

**Issue by chrisbouchard** [neovide/neovide#181]
I had my `guifont` set in `${nvim_config_dir}/ginit.vim`, but Neovide didn't pick it up until I moved the line to `init.vim`. I'm not sure if it's a Neovim standard file, but it seems to be standard among other GUI clients I've used.

Also (possibly a separate issue?), it would be nice to have a variable I could check to detect if I'm running in Neovide. E.g., [Neovim-Gtk uses](https://github.com/daa84/neovim-gtk/wiki/Configuration)

```vim
if exists('g:GtkGuiLoaded')
  " some code here
endif
```

**Comment by chrisbouchard** [comment]
Also, is there a way to set the inintial lines and columns? I tried adding

```
set lines=40
set columns=120
```

in my `init.vim`, but that doesn't seem to get picked up, and setting them after launch doesn't resize the window.

**Comment by Kethku** [comment]
Just pushed a fix which loads ginit.vim
Neovide declares a global variable "g:neovide" which you can use like you say

I'm not sure how lines and columns setting should work... Let me think about it

**Comment by chrisbouchard** [comment]
Thanks for being so responsive! Neovide is the best Neovim GUI I've used yet, so I'm interested in making it my daily driver.

**Comment by ABitMoreDepth** [comment]
How does one set the font and size in neovide?

**Comment by chrisbouchard** [comment]
@Kethku commented [in a different issue](https://github.com/Kethku/neovide/issues/146#issuecomment-589236024):

```
set guifont=Delugia\ Nerd\ Font:h12
```

**Comment by ABitMoreDepth** [comment]
(I'm a bit blind at times methinks!)


Follow up question, perhaps for its own issue, does Ctrl+6 correctly swap buffers for you?  It works for me in neovim and neovim-gtk, but not in neovide?

**Comment by svermeulen** [comment]
I'm on commit ef2d99b and I still find that the ginit.vim does not seem to be loaded.  I've had to move my neovide config to init.vim as well

**Comment by svermeulen** [comment]
Tried this again with the latest (36e3040) and ginit.vim still does not appear to be loaded

**Comment by nkarl** [comment]
@svermeulen which system are you running on? The `ginit.vim` file should be automatically loaded provided that you put it in the same folder as your `init.vim`. Try set the font size to something outrageous like 20 or 30 and reload your terminal before starting neovide again:

```set guifont=FiraCode\ Nerd\ Font:h20```

**Comment by anuvyklack** [comment]
On 0b976c3 the ginit.vim still doesn't loaded.

**Comment by Kethku** [comment]
That is a separate issue. I will create it.
